src/screens/HomeScreen.tsx

```diff
@@ -54,3 +59,28 @@ const styles = StyleSheet.create({
     flex: 1,
   },
 });
+
+function TrackTileWrapper() {
+  const navigation = useNavigation<HomeScreenNavigation>();
+  const { data: appConfig } = useAppConfig();
+
+  // TODO: Push this logic to the useAppConfig hook
```

Comment:
RE: this TODO: I'd like the `useAppConfig` hook to stay dumb / API-driven if possible.  If there are props like these we need on top of the API response, we should consider creating a hook that's layered on top of `useAppConfig`.  In this case, it could be something like `useTrackTileConfig()`.  If the props are only used in one place, IMO the hook isn't needed and it's fine for the props to just be created where they're used.

Reply:
I like this idea a lot!   Will do.